code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm

```diff
@@ -185,3 +218,125 @@
 	if(!can_use_ability(target, override_flags = XACT_IGNORE_SELECTED_ABILITY))
 		return ..()
 	return TRUE
+
+
+
+// ***************************************
+// *********** Rage
+// ***************************************
+/datum/action/xeno_action/activable/rage
+	name = "Rage"
+	action_icon_state = "rage"
+	mechanics_text = "Use while at 50% health or lower to gain extra slash damage, resistances and speed in proportion to your missing hit points. This bonus is increased and you regain plasma while your HP is negative."
+	ability_name = "Rage"
+	plasma_cost = 0 //We're limited by cooldowns, not plasma
+	cooldown_timer = 120 SECONDS
+	keybind_flags = XACT_KEYBIND_USE_ABILITY | XACT_IGNORE_SELECTED_ABILITY
+	keybind_signal = COMSIG_XENOABILITY_RAGE
+	var/rage_power
+
+/datum/action/xeno_action/activable/rage/on_cooldown_finish()
+	to_chat(owner, "<span class='xenodanger'>We are able to enter our rage once again.</span>")
+	owner.playsound_local(owner, 'sound/effects/xeno_newlarva.ogg', 25, 0, 1)
+	return ..()
+
+/datum/action/xeno_action/activable/rage/can_use_ability(atom/A, silent = FALSE, override_flags)
+	. = ..()
+
+	var/mob/living/carbon/xenomorph/ravager/rager = owner
+
+	var/rage_health_threshold = rager.maxHealth * 0.5 //Need to be at 50% of max hp or lower to rage
+	if(rager.health > rage_health_threshold)
+		if(!silent)
+			to_chat(rager, "<span class='xenodanger'>Our health isn't low enough to rage! We must take [rager.health - rage_health_threshold] more damage!</span>")
+		return FALSE
+
+
+/datum/action/xeno_action/activable/rage/use_ability(atom/A)
+	var/mob/living/carbon/xenomorph/ravager/X = owner
+
+	rage_power = (1-(X.health/X.maxHealth)) * 0.5 //Calculate the power of our rage; scales with difference between current and max HP
+
+	if(X.health < 0) //Double dip on the negative HP; gain + 0.005 rage power per point of negative HP
+		rage_power += X.health * -0.01 * 0.5
```

Comment:
```suggestion
		rage_power += X.health * -0.005
```

Reply:
Yeah fair, I was tooling around with values earlier so this should be done now that they're finalized